crates/forge_infra/src/forge_infra.rs

```diff
@@ -64,6 +68,7 @@ impl Infrastructure for ForgeInfra {
     type CommandExecutorService = ForgeCommandExecutorService;
     type InquireService = ForgeInquire;
     type McpServer = ForgeMcpServer;
+    type IndexerService = Indexer;
```

Comment:
Infra should contain things that we can swap-in/out. For eg: Storage, Vector Index, Embedding Providers etc.